src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/placeholders/PlaceholderConstants.java

```diff
@@ -6,13 +6,16 @@ public final class PlaceholderConstants {
     public static final String WELSH_CASE_CREATED_DATE_LITERAL = "welsh_case_created_date";
     public static final String BENEFIT_TYPE_LITERAL = "benefit_type";
     public static final String SHOULD_HIDE_NINO = "should_hide_nino";
+    public static final String SHOULD_HIDE_IBCA_REFERENCE = "should_hide_ibca_reference";
     public static final String APPELLANT_FULL_NAME_LITERAL = "appellant_full_name";
     public static final String CASE_ID_LITERAL = "case_id";
     public static final String NINO_LITERAL = "nino";
+    public static final String IBCA_REFERENCE_LITERAL = "ibca_reference";
     public static final String GENERATED_DATE_LITERAL = "generated_date";
     public static final String WELSH_GENERATED_DATE_LITERAL = "welsh_generated_date";
     public static final String SSCS_URL_LITERAL = "sscs_url";
     public static final String SSCS_URL = "https://www.gov.uk/appeal-benefit-decision";
+    public static final String IBCA_URL = "www.gov.uk/guidance/appeal-an-infected-blood-compensation-decision";
```

Comment:
not need the https:// ?

Reply:
Yeah good shout. I just took the value from the ticket. Let me look into this